docs/fairlearn.exceptions.rst

```diff
@@ -1,7 +0,0 @@
-fairlearn\.exceptions module
-----------------------------
-
-.. automodule:: fairlearn.exceptions
```

Comment:
Where did the contents go?

Reply:
This isn't a directory and is automatically included. I got a warning saying it's duplicate. If we want it to be its own module I can move it into a directory. What do you think?

Reply:
It is a producing output, though:
https://fairlearn.readthedocs.io/en/latest/fairlearn.exceptions.html

Reply:
Oh I see why I got that impression. readthedocs doesn't show the top-level module with the class. Anyway, not a major thing, I'll leave it unchanged (i.e. revert my change)